src/main/java/com/thedariusz/warnme/twitter/model/Meta.java

```diff
@@ -0,0 +1,51 @@
+package com.thedariusz.warnme.twitter.model;
+
+public class Meta {
+
+    private String oldestId;
+    private String newestId;
+    private String resultCount;
+    private String nextToken;
+
+    public String getOldestId() {
+        return oldestId;
+    }
+
+    public void setOldestId(String oldestId) {
+        this.oldestId = oldestId;
+    }
+
+    public String getNewestId() {
+        return newestId;
+    }
+
+    public void setNewestId(String newestId) {
+        this.newestId = newestId;
+    }
+
+    public String getResultCount() {
+        return resultCount;
+    }
+
+    public void setResultCount(String resultCount) {
+        this.resultCount = resultCount;
+    }
+
+    public String getNextToken() {
+        return nextToken;
+    }
+
+    public void setNextToken(String nextToken) {
+        this.nextToken = nextToken;
+    }
+
+    public Meta(String oldestId, String newestId, String resultCount, String nextToken) {
+        this.oldestId = oldestId;
+        this.newestId = newestId;
+        this.resultCount = resultCount;
+        this.nextToken = nextToken;
+    }
+
+    public Meta() {
+    }
```

Comment:
constructors should be on top of the class. 
1. static fields
2. static block
3. fields
4. static methods
4. constructors
5. private methods
6. getters and setters